hawc/apps/vocab/views.py

```diff
@@ -12,16 +11,12 @@ class EhvBrowse(TemplateView):
 
     def _get_config(self) -> str:
         # get EHV in json; use cache if possible
-        key = "ehv-dataframe-json"
-        data = cache.get(key)
-        if data is None:
-            data = WebappConfig(
-                app="animalStartup",
-                page="ehvBrowserStartup",
-                data={"data": models.Term.ehv_dataframe().to_csv(index=False)},
-            ).json()
-            cache.set(key, data, settings.CACHE_10_MIN)
-        return data
+        data = WebappConfig(
```

Comment:
all the others look good, but this will be evaluated every time regardless of the cache status; you could either put this in a lambda or put in a function